classes/MyAdmin.php

```diff
@@ -342,10 +360,8 @@ protected function outputAdminAgendas()
     protected function outputAdminBodyEnd()
     {
         $TableAdmin = $this->TableAdmin;
-        $result = //<script type="text/javascript" src="https://code.jquery.com/jquery.js"></script>
-            '<script type="text/javascript" src="scripts/jquery.js"></script>'
-            . '<script type="text/javascript" src="scripts/popper.js"></script>'
-            . '<script type="text/javascript" src="scripts/bootstrap.js"></script>'
+        $result = 
+            (empty($this->javascripts) ? '' : ('<script type="text/javascript" src="' . implode('"></script><script type="text/javascript" src="', $this->javascripts) . '"></script>' ))
             //<script type="text/javascript" src="scripts/bootstrap-datetimepicker.js"></script>
             . '<script type="text/javascript" src="scripts/jquery.sha1.js"></script>'
             . '<script type="text/javascript" src="scripts/summernote.js"></script>'
```

Comment:
Částečně už's to tak načal, ale.. co bys říkal na to mít u MyAdmin.php attributy (typu pole) pro style-sheety a javascripty, které by si MyAdmin.__construct() vyplnil soubory, které potřebuje admin v základní verzi, s tím, že jeho potomci by si do konstruktoru mohli přidat další klientské soubory/knihovny?
A v outputAdminHead() by pak byla deklarace těch souborů přes obyčejné foreach()?
Dokázal bych si představit i jedno asociativní pole, kde by kromě cesty k souboru byl i jeho typ, který by se ve foreach správně vyplnil (zatím máme jen <link rel="stylesheet"..> pro css a <script type="text/javascript" charset="utf-8"...> pro JavaScript). Pro potomky MyAdmin.php by metoda outputAdminHead() zůstala stejná - tj. i při přidání nové knihovny by ho programátor nemusel přepisovat.